modules/programs/fish.nix

```
@@ -285,6 +285,8 @@ in {
     {
       home.packages = [ cfg.package ];
 
+      programs.man.generateCaches = true;
```

Sorry for the duplicate review. I think we should only set a default value for `programs.man.generateCaches` so the user can override it without using `lib.mkOverride`. (This is what Nixpkgs currently does.) We might as well include a comment to clarify what generating the caches does too.
```suggestion
      # Required for man completions
      programs.man.generateCaches = mkDefault true;
```

---

Thanks. `mkDefault` is added. I prefer to explain the reason of a commit in its commit message though.